setup.py

```diff
@@ -9,6 +9,6 @@
     install_requires=["beautifulsoup4", "click", "requests"],
     entry_points="""
         [console_scripts]
-        festerize=festerize:cli
```

Comment:
FYI, the format of this string is <script-name>=<python-module>:<function-name>. The click docs sometimes use `cli` as the method name, and sometimes they use the same name as the enclosing module. The choice is arbitrary but I thought the latter looks a bit better.